### DISCUSSION TITLE
1806 |       #error "Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN requires Z_MIN_ENDSTOP_INVERTING to match Z_MIN_PROBE_ENDSTOP_INVERTING."

### DISCUSSION THREAD

**Discussion by MaynardMann** [mriscoc/Ender3V2S1#368] [Q&A]
trying to compile a UBL firmware for s1 used the configurator to generate configs but auto build marlin is kicking back this error when compiling the bin

![image](https://user-images.githubusercontent.com/60556984/189768530-43a96461-1bd7-4f32-b3ef-64f32f8d9749.png)

**Comment by mriscoc** [comment]
You found a bug, please re-download the special configuration repository or make this local change:
![image](https://user-images.githubusercontent.com/2745567/189781934-8109adba-8dd7-4f61-8415-904159e5dff8.png)

**Reply by MaynardMann** [reply]
Hmm no error on that front now now I have another issue even with just able and ubl selected no additional features auto build marlin is saying it won't fit on the board and I'm not sure whatelse I can strip away. Is there any hope for f4 users to get UBL onto the bourd with Abl selected?  


![image](https://user-images.githubusercontent.com/60556984/189784422-f3b3e305-c5b3-4d9b-b993-8636eddf12be.png)

**Reply by mriscoc** [reply]
That is because the F4 program memory is full, you cannot use UBL without disabling some features. That's why I don't give F4 UBL versions.
